go.mod

```diff
@@ -2,7 +2,10 @@ module github.com/tektoncd/chains
 
 go 1.16
 
-replace k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20211109043538-20434351676c
+replace (
+	github.com/sigstore/cosign => github.com/sigstore/cosign v1.5.2-0.20220208171233-ddca5deac2ba
```

Comment:
Do we actually need a `replace` or are you using this to fast-forward to HEAD?

Reply:
Ok, we shouldn't need a replace to pull in the newer version.  This should do it:
```
go get -d github.com/sigstore/cosign@main
./hack/update-deps.sh
```

Reply:
That's right! Spaced, thanks! Done